### ISSUE TITLE
iOS StopListening() After 3 Seconds

### ISSUE THREAD

**Issue by M-Yusuf** [pbakondy/cordova-plugin-speechrecognition#34]
Is it possible to have speechRecognition() only listen for 3 seconds in iOS?

**Comment by aurothic** [comment]
Yes, you can use `setTimeout` to call `SpeechRecognition.stopListening()` after so much time as passed.
